### PR TITLE
functional-tester: use different ports in Procfile

### PR DESCRIPTION
**Pull request by gyuho**
@heyitsanthony @fanminshi 

Current `Procfile` doesn't work because of port conflict.

### CODE REVIEW

**Comment by xiang90** [comment]
Can we change procfile instead? or we have to change our prom deployment.

**Comment by gyuho** [comment]
You are right. Just fixed. Thanks.

File: tools/functional-tester/Procfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-agent-1: mkdir -p agent-1 && cd agent-1 && ../bin/etcd-agent -etcd-path ../bin/etcd -port localhost:9027 -use-root=false
-agent-2: mkdir -p agent-2 && cd agent-2 && ../bin/etcd-agent -etcd-path ../bin/etcd -port localhost:9028 -use-root=false
-agent-3: mkdir -p agent-3 && cd agent-3 && ../bin/etcd-agent -etcd-path ../bin/etcd -port localhost:9029 -use-root=false
-stresser: sleep 1s && bin/etcd-tester -agent-endpoints "localhost:9027,localhost:9028,localhost:9029"  -client-ports 12379,22379,32379 -peer-ports 12380,22380,32380
+agent-1: mkdir -p agent-1 && cd agent-1 && ../bin/etcd-agent -etcd-path ../bin/etcd -port localhost:8027 -use-root=false
```
Comment (xiang90) [inline tools/functional-tester/Procfile]: how about 19027, 29027 and 39037?

Comment (gyuho) [inline tools/functional-tester/Procfile]: Just fixed. Thanks.

Comment (xiang90) [inline tools/functional-tester/Procfile]: I want 19027, 29027 and 39037, not localhost:19027,localhost:19028,localhost:19029.

I want to make this consistent with the etcd addr:
```
12379,22379,32379 
```

Comment (gyuho) [inline tools/functional-tester/Procfile]: right, fixed. Thanks.

**Comment by xiang90** [comment]
lgtm

**Pull request merged**